src/doc/rustc/src/platform-support.md

```diff
@@ -220,7 +220,7 @@ target | std | host | notes
 `armv5te-unknown-linux-uclibceabi` | ? |  | ARMv5TE Linux with uClibc
 `armv6-unknown-freebsd` | ✓ | ✓ | ARMv6 FreeBSD
 `armv6-unknown-netbsd-eabihf` | ? |  |
-`armv6k-nintendo-3ds` | * |  | ARMv6K Nintendo 3DS, Horizon (Requires devkitARM toolchain)
+[`armv6k-nintendo-3ds`](platform-support/armv6k-nintendo-3ds.md) | * |  | ARMv6K Nintendo 3DS, Horizon (Requires devkitARM toolchain)
```

Comment:
Oops, I noticed this the other day but forgot to mention, I guess we will want to change `*` to `?` here, based on its meaning for the Tier 3 table:

> The `std` column in the table below has the following meanings:
> 
> * ✓ indicates the full standard library is available.
> * \* indicates the target only supports [`no_std`] development.
> * ? indicates the standard library support is unknown or a work-in-progress.
>